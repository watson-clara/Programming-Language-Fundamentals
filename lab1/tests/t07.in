debug off
stopwords on
load tracks.txt
filter title
filter stopwords
count tracks
count words
count characters
