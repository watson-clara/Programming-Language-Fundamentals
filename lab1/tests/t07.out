100000
299959
1917399
